runfolder_tests/integration/rest_tests.py

```diff
@@ -27,8 +45,13 @@ def line_count(path):
                 count += 1
             return count
 
-        self.messages_logged = TestFunctionDelta(
-            lambda: line_count('./runfolder.log'), self, 0.1)
+        # Disable the log check if not running locally:
+        if not self._base_url().startswith("http://localhost"):
```

Comment:
Yes, it's messy because the tests are now run in two different scenarios, remote and locally. 

When running locally, I threw in a few tests to see if the logger is working as expected (just so we would have that tested somewhere - not the idea to do that in all integration tests). But when running remotely, I'm not going to run those tests.

It would be clearer to just have the log tests elsewhere and not run those in remote mode, but this was a quick fix.